app/docs/homepage.stories.tsx

```diff
@@ -33,12 +46,53 @@ const HomepageStory = {
           example2Headline="Use powerful customizations"
           example2Description="With the help of custom filters and data segmentation, even complex issues can be visualized."
         />
-        <Contribute
-          headline="Would you like to visualize your own data?"
-          description="Find out how you can integrate your data into the LINDAS Linked Data Service."
-          buttonLabel="Learn how"
-          buttonUrl="https://lindas.admin.ch/?lang=en"
-        />
+        <Section>
+          <Contribute
+            headline="Would you like to visualize your own data?"
+            description="Find out how you can integrate your data into the LINDAS Linked Data Service."
+            buttonLabel="Learn how"
+            buttonUrl="https://lindas.admin.ch/?lang=en"
+          />
+          <div style={{ width: "1px", backgroundColor: "#e5e5e5" }}></div>
+          <Newsletter
+            headline="Subscribe to our Newsletter"
+            description="Stay up to day and subscribe to our newsletter by adding your email address below."
+            buttonLabel="Subscribe"
+            buttonUrl="mailto:visualize@bafu.admin.ch"
+          />
+        </Section>
+
+        <Section
+          sx={{
+            color: "gray.800",
+          }}
+        >
+          <BugReport
+            headline="Found a bug?"
+            description="Please report the bug, so can fix it as soon as possible"
```

Comment:
I think we miss a dot at the end here? I also wonder if this shouldn't come from Accent, so Kerstin could modify the texts. But it could be a bigger thing and we didn't use it already for these pages, so just a thought 👍

Reply:
yeah, generally speaking this is something to consider when implementing the new Landing Page.